sys/include/net/gcoap_helper.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gcoap_helper GCoAP helper functions
+ * @ingroup     net_gcoap
```

Comment:
If its not intended to be its own module, maybe use
```suggestion
 * @addtogroup  net_gcoap
```
instead?